Common/src/CConfig.cpp

```diff
@@ -2051,7 +2051,7 @@ void CConfig::SetConfig_Options() {
 
   /*!\par CONFIG_CATEGORY: Input/output files and formats \ingroup Config */
   /*--- Options related to input/output files and formats ---*/
-
+nijso
```

Comment:
copy-paste mistake or reminder?

Reply:
this is where I am with checking the config options :-)